src/ontology/mondo-edit.obo

```diff
@@ -570175,6 +570185,3049 @@ intersection_of: MONDO:0700097 MONDO:0000721 ! cross-species analog xanthinuria
+xref: OMIA:000835 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000835"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:000835", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011693
+name: reduced glutathione deficiency due to GCS deficiency, non-human animal
+xref: OMIA:000853 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000853"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:000853", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011694
+name: thrombasthenia, non-human animal
+xref: OMIA:001000 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001000"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:001000", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011695
+name: thrombopathia, non-human animal
+xref: OMIA:001003 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001003"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:001003", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011696
+name: stomatocytosis and gastrits, non-human animal
+xref: OMIA:001178 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001178"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:001178", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011697
+name: hemophagocytic syndrome, non-human animal
+xref: OMIA:001229 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001229"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:001229", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011698
+name: reduced glutathione deficiency due to amino-acid transport defect, non-human animal
+xref: OMIA:001234 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001234"} ! non-human animal disease
+is_a: MONDO:1011319 {source="OMIA:001234", source="https://orcid.org/0000-0002-5002-8648"} ! hematologic disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
```

Comment:
could this be also a child of MONDO:1011699; reduced glutathione deficiency, non-human animal ?

Reply:
done also made 'reduced glutathione deficiency due to GCS deficiency, non-human animal' a a child of MONDO:1011699; reduced glutathione deficiency, non-human animal